### PR TITLE
 add i18n Australia(en_AU)

### PR DESCRIPTION
**Pull request by DLErice**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [x] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by DLErice** [comment]
add en_AU for Australia

**Comment by yoyo837** [comment]
Please refer to #31346

**Comment by yoyo837** [comment]
`feature` branch pls.

**Comment by DLErice** [comment]
add en_AU

File: components/locale/en_AU.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import defaultLocale from './default';
+
+export default defaultLocale;
```
Comment (afc163) [inline components/locale/en_AU.tsx:3]: Is it en_AU completely same as en_US?

Comment (afc163) [reply components/locale/en_AU.tsx:3]: Closing for no response, feel free to reply it.

**Pull request closed without merging**